resources/todomvc/architecture-examples/angular-complex/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "angular-complex",
+    "version": "1.0.0",
+    "description": "TodoMVC complex app written with Angular.",
+    "private": true,
+    "engines": {
+        "node": ">=18.13.0",
+        "npm": ">=8.19.3"
+    },
+    "scripts": {
+        "build": "node scripts/build.js",
+        "serve": "http-server ./dist -p 7002 -c-1 --cors"
+    },
+    "devDependencies": {
+        "angular": "file:../angular",
```

Comment:
+1

Reply:
it could be something like:
```
todomvc-angular
todomvc-angular-complex

todomvc-es5
todomvc-es5-complex
```

what do you think @julienw @issackjohn ?

Reply:
SGTM. There aren't plans to publish these packages to npm right? I see `todomvc-react` already published there.

Reply:
no plans to publish to npm.

Reply:
> it could be something like:
> 
> ```
> todomvc-angular
> todomvc-angular-complex
> 
> todomvc-es5
> todomvc-es5-complex
> ```
> 
> what do you think @julienw @issackjohn ?

sounds good to me

Reply:
Also the build scripts rely on the fact that the standalone versions have been built with their latest source. I'm not sure we should assume this, and instead we could run `npm i && npm run build` on the related standalone version first.

Reply:
using a local install vs just pointing to the local folder doesn't bother me. I think that's consistent with the way we use the todo-css and news-css folders currently. 

Regarding running the build script before, I think that's an option to force a rebuild before... there are probably reasons for either way. It would be nice if we'd be using pnpm, cause then we could install a local package by version 🤷

Reply:
It makes sense for the other cases because we're using `import` or `require` on them (as far as I remember), but here we're merely copying the files.